index.d.ts

```diff
@@ -102,22 +101,18 @@ export interface Instance {
   }
 }
 
-export interface Collection {
-  destroyAll(): void
-  instances: Instance[]
-  props: Props
-  targets: Targets
+export interface HideAllOptions {
+  exclude?: Instance
+  checkHideOnClick?: boolean
+  duration?: number
 }
 
 export interface Tippy {
-  (targets: Targets, options?: Props): Collection
+  (targets: Targets, options?: Props): Instance | Instance[]
   readonly defaults: Props
   readonly version: string
-  disableAnimations(): void
-  hideAllPoppers(): void
-  one(targets: Targets, options?: Props): Instance
+  hideAll(options: HideAllOptions): void
```

Comment:
```suggestion
  hideAll(options?: HideAllOptions): void
```
Otherwise, it will require an object to be passed.